src/app/search-results/page.tsx

```diff
@@ -0,0 +1,68 @@
+"use client";
+
+import './page.scss';
+
+import { useEffect } from 'react';
+import { SafDivider } from '@saffron/core-components/react';
+
+import { AppsFilter } from 'app/components/shared/AppsFilter/AppsFilter';
+import { useAppStoreSearch } from 'app/components/shared/AppStoreSearch/useAppStoreSearch';
+import { useQueryFilters } from 'app/shared/hooks/useQueryFilters';
+import { useSearchAutocomplete } from 'app/components/shared/SafSearchAutocomplete/useSearchAutocomplete';
+import { useAppStoreQuery } from 'app/components/shared/AppStoreSearch/useAppStoreQuery';
+import { useGlobalLoader } from 'app/shared/hooks/useGlobalLoader';
+import { AppStoreCardSearchResults } from 'app/components/shared/AppStoreSearch/AppStoreCardSearchResults';
+
+export function SearchPage() {
+    const { apps, searchApps } = useAppStoreSearch();
+
+    const { sortOption, sortOrder, queryFilters } = useQueryFilters();
+
+    const { inputValue: query } = useSearchAutocomplete();
+
+    const { updateQueryValue } = useAppStoreQuery();
+
+    const { isLoading } = useGlobalLoader();
+
+    // When loading the page in the navigator using the url we need to rehydrate state
+    useEffect(() => {
+        updateQueryValue(query);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
+
+    useEffect(() => {
+        searchApps({
+            query,
+            queryFilters,
+            sortOption,
+            sortOrder
+        });
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [query, queryFilters, sortOption]);
+
+    if (isLoading) {
+        return null;
+    }
+
+    return (
+        // TODO: Is it necessary to add a base layout??
+        <div className='p-4 apps-page search-page' data-testid='search-page'>
+            {/* <HelmetTags title='Search results' /> */}
```

Comment:
Could you remove this and the Nextjs metadata? This seems to be static, so it could be in a layout file. You can check an example on src/app/apps/layout.tsx

Reply:
Metadata file is already here 👍 

src/app/search-results/layout.tsx